.github/workflows/aws-integration-test.yml

```diff
@@ -0,0 +1,139 @@
+permissions:
+  contents: read
+name: LocalStack AWS Integration Test
+on:
+  workflow_call:
```

Comment:
```suggestion
  workflow_call:
  workflow_dispatch:
```

The `README` includes a mention of the extra trigger. Might as well add it as a possibility.

Reply:
Will do. I'll add some more info around the recent changes as well. Thanks for reviewing this.